gnovm/pkg/gnolang/eval_test.go

```diff
@@ -88,7 +106,10 @@ func commentFrom(s string, delims []string) []string {
 			next = directivesFound[i+1].index
 		}
 
-		directivesFound[i].res = strings.TrimSpace(strings.ReplaceAll(s[directivesFound[i].index+len(directivesFound[i].delim):next], "\n// ", "\n"))
+		parsed := reCommentPrefix.ReplaceAllLiteralString(
```

Comment:
I know this was a long line of code before you change, but a comment here to say what this does would go a long way for readability.

Reply:
Split up a bit and added some comments :heavy_check_mark: